Jetcaster/wear/src/main/java/com/example/jetcaster/ui/podcast/PodcastDetailsScreen.kt

```diff
@@ -0,0 +1,244 @@
+        onPlayEpisode = podcastDetailsViewModel::onPlayEpisode,
+        onErrorDialogCancelClick = onErrorDialogCancelClick,
+        onPlayButtonClick = onPlayButtonClick
+    )
+}
+
+@Composable
+fun PodcastDetailsScreen(
+    viewState: PodcastDetailsScreenState,
+    onChangeSpeedButtonClick: () -> Unit,
+    onPlayButtonClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    onEpisodeItemClick: (EpisodeToPodcast) -> Unit,
+    onPlayEpisode: (PlayerEpisode) -> Unit,
+    onErrorDialogCancelClick: () -> Unit
+) {
+    val columnState = rememberColumnState()
+    ScreenScaffold(
+        scrollState = columnState,
+        modifier = modifier
+    ) {
+        when (viewState) {
+            is PodcastDetailsScreenState.Loaded -> {
+                EntityScreen(
+                    columnState = columnState,
+                    headerContent = { DefaultEntityScreenHeader(title = viewState.podcast.title) },
+                    buttonsContent = {
+                        ButtonsContent(
+                            episodes = viewState.episodeList,
+                            onChangeSpeedButtonClick = onChangeSpeedButtonClick,
+                            onPlayButtonClick = onPlayButtonClick,
+                            onPlayEpisode = onPlayEpisode
+                        )
+                    },
+                    content = {
+                        items(count = viewState.episodeList.size) { index ->
+                            MediaContent(
+                                episode = viewState.episodeList[index],
+                                downloadItemArtworkPlaceholder = rememberVectorPainter(
+                                    image = Icons.Default.MusicNote,
+                                    tintColor = Color.Blue,
+                                ),
+                                onEpisodeItemClick
+                            )
+                        }
+                    }
+                )
+            }
+
+            PodcastDetailsScreenState.Empty,
+            PodcastDetailsScreenState.Loading -> {
+                Column {
+                    PlaceholderChip(colors = ChipDefaults.secondaryChipColors())
+                }
+            }
+        }
+    }
+    Dialog(
+        showDialog = viewState == PodcastDetailsScreenState.Empty,
+        onDismissRequest = onErrorDialogCancelClick,
+        scrollState = rememberScalingLazyListState(),
+    ) {
+        Alert(
+            title = {
+                Text(
+                    text = stringResource(R.string.podcasts_no_episode_podcasts),
+                    color = MaterialTheme.colors.onBackground,
+                    textAlign = TextAlign.Center,
+                    style = MaterialTheme.typography.title3,
+                )
+            },
+        ) {
+            item {
+                Row(
+                    verticalAlignment = Alignment.CenterVertically,
+                ) {
+
+                    Button(
```

Comment:
AlertDialog has slots for buttons, you should let it create the buttons. It will be responsive to display size

![image](https://github.com/android/compose-samples/assets/231923/08a87acc-e58b-424a-a09c-ca12794d7b89)

Reply:
thanks, will update on the Media Toolkit as well